app/screens/ScheduleScreen/ScheduleScreen.tsx

```diff
@@ -196,15 +199,17 @@ export const ScheduleScreen: React.FC<TabScreenProps<"Schedule">> = () => {
               <ScheduleCard
                 {...item}
                 isPast={
-                  index < scheduleIndex || (index === scheduleIndex && itemIndex < eventIndex)
+                  index < scheduleIndex ||
+                  (index === scheduleIndex && itemIndex < eventIndex) ||
+                  isConfOver
```

Comment:
I noticed that the card loses its style to show if it's the past event after the conference is done since it's not getting any event index to reference. So changed the logic AGAIN!